sdk/program/src/loader_upgradeable_instruction.rs

```diff
@@ -46,10 +46,12 @@ pub enum UpgradeableLoaderInstruction {
     /// The ProgramData address is derived from the Program account's address as
     /// follows:
     ///
-    /// `let (program_data_address, _) = Pubkey::find_program_address(
+    /// ```
```

Comment:
```suggestion
    /// ```no_run
```
Looks like this PR will need a smattering of these until we invest time into nicer doc tests

Reply:
@brson would you like us to pick this change up from where you left off?

Reply:
Really appreciate the clean up you've done so far btw 😄

Reply:
@jstarry I pushed a commit that fixes the test so it works. Thanks for the review.